src/app/shared/navigation-focus/navigation-focus.ts

```diff
@@ -1,49 +1,23 @@
 import {NgModule, Directive, ElementRef, HostBinding, OnDestroy} from '@angular/core';
-import {Event, Router, NavigationEnd} from '@angular/router';
-import {filter} from 'rxjs/operators';
-import {Subscription} from 'rxjs';
-
-/** The timeout id of the previous focus change. */
-let lastTimeoutId = -1;
+import {NavigationFocusService} from './navigation-focus.service';
 
 @Directive({
   selector: '[focusOnNavigation]',
 })
 export class NavigationFocus implements OnDestroy {
   @HostBinding('tabindex') role = '-1';
```

Comment:
I haven no idea why the property is called `role`, can you change it to `tabindex`